### DISCUSSION TITLE
End of April shutdown checklist

### DISCUSSION THREAD

**Discussion by bsweger** [reichlab/covid19-forecast-hub#9274] [General]
Using this issue as a place to keep track of tasks related to shutting down the COVID hub at the end of April.

- [x] Turn off routine actions and things happening on AWS related to data and model builds
    - [x] truth data actions in the hub
    - [x] model runs
    - [x] viz update action (Evan)
    - [x] actions running in covidData (can probably turn these off prior to official shutdown)
- [x] One-time full rebuild of the hub viz to ensure all forecasts show up (Evan)
- [x] Actions putting data into Zoltar
    - [x] covid
    - [x] flu
- [ ] text on covid19forecast hub website (Alvaro and Nick)
- [x] One-time sync of all COVID-19 forecast data to Zoltar (Matt) - https://github.com/reichlab/covid19-forecast-hub/actions/workflows/upload_to_zoltar.yml
- [x] Review AWS EFS usage, make recommendation (Matt)

**Comment by matthewcornell** [comment]
For reference: As of 2024-03-06, these are the current [AWS script schedule and approximate timings (R=EventBridge Rule, S=EventBridge Scheduler, x=disabled)](https://reichlab.slack.com/archives/C02SJFAT7NK/p1663000383956199). Let's check those that should be disabled at the end of April. (They can be easily disabled without deleting. )

- [x] Mon 9a EST:    baseline-model (S) | ~5 min
- [x] Mon 9.30a EST: sarix-model (S) | ~2:45 hr:min
- [x] Mon 1p EST:    run-covid19-hosp-trend-ensemble.sh (R) | ~45 min
- [x] Mon 3.15p EST: ensemble-model (S) | ~15 min
- [x] Mon 8p EST:    run-hosp-forecast-model-filters.sh (R) | ~1 min
- [x] x Tue 9.30a:     run-weekly-reports.sh (R) | ~15 min
- [x] Tue 12n EST:   deploy-covidhub-site.sh (R) | ~15 min
- [x] Tue 3p EST:    deploy-covidhub-site.sh (R) | ~15 min
- [x] Thu 12n EST:   flusight-eval (S) | ~11 min

**Reply by elray1** [reply]
I think we can basically disable all of these, but I left the following ones unchecked:

- I don't know what run-hosp-forecast-model-filters.sh does
- I also don't know what deploy-covidhub-site.sh does other than what i can infer from the name... it seems like we will not need to redeploy the website routinely though.
- We will want to be sure that the flusight-eval job runs once more on May 2

**Reply by matthewcornell** [reply]
@elray1 : Details about those two scripts:

- [run-hosp-forecast-model-filters.sh](https://github.com/reichlab/covidModels/blob/master/aws-vm-scripts/run-hosp-forecast-model-filters.sh) : knits [hospital-model-forecaster-filtering.Rmd](https://github.com/reichlab/covid19-forecast-hub/blob/master/code/reports/hospital-model-forecaster-filtering.Rmd) and publishes to https://covid19forecasthub.org/hospital-model-forecaster-filtering.html . ("Goal: Identify hospitalization forecast models that were excluded from the CDC 4 week ensemble due to submitting forecasts for less than 28 days for the current week."
- [deploy-covidhub-site.sh](https://github.com/reichlab/covidModels/blob/master/aws-vm-scripts/deploy-covidhub-site.sh): builds and pushes https://github.com/reichlab/covid19-forecast-hub-web

Re: **We will want to be sure that the flusight-eval job runs once more on May 2**: Our meeting is on Thu 5/2/024 at 2p EST, and the script runs at 12n EST, so we should be ok, correct?

**Reply by elray1** [reply]
r.e. timing, i think we should be ok.  Last week, that run took 2 hours to complete.  And anyways, all we will be doing is turning off scheduling for future runs, right?

**Reply by matthewcornell** [reply]
Correct.
